src/goalServices/closeMultiRecipientGoalsFromAdmin.test.js

```diff
@@ -157,18 +157,10 @@ describe('closeMultiRecipientGoalsFromAdmin', () => {
 
     updatedGoals.forEach((updatedGoal) => {
       expect(updatedGoal.status).toBe(GOAL_STATUS.CLOSED);
-      expect(updatedGoal.closeSuspendContext).toBe(data.closeSuspendContext);
-      expect(updatedGoal.closeSuspendReason).toBe(data.closeSuspendReason);
       updatedGoal.objectives.forEach((objective) => {
         const expectedStatus = objective.onApprovedAR
           ? OBJECTIVE_STATUS.COMPLETE : objectiveNotOnApprovedAr.status;
-        const expectedSuspendReason = objective.onApprovedAR
-          ? data.closeSuspendReason : null;
-        const expectedSuspendContext = objective.onApprovedAR
-          ? data.closeSuspendContext : null;
         expect(objective.status).toBe(expectedStatus);
-        expect(objective.closeSuspendReason).toBe(expectedSuspendReason);
-        expect(objective.closeSuspendContext).toBe(expectedSuspendContext);
```

Comment:
Removed these assertions because closeSuspendReason and closeSuspendContext are *not* going to be stored on the `Goal` any more.